src/app/modules/checkout/components/cart-summary/cart-summary.component.html

```diff
@@ -0,0 +1,50 @@
+<div class="component">
+    <div class="card mb-3">
+      <h3 class="card-header">Order Summary</h3>
+      <div class="card-body">
+        <div class="row pb-3">
+            <div class="col">
+                <p class="summary-item">Subtotal</p>
+              </div>
+              <div class="col">
+                <p class="summary-value">{{ cart?.subTotal?.formattedAmount }}</p>
```

Comment:
No default values?

Reply:
this is a xapi response
![Screenshot_36](https://user-images.githubusercontent.com/20122385/127635554-3b9c93e2-99d5-49e2-bfb7-e63b97cfdd0b.png)